src/main/java/ch/uzh/ifi/hase/soprafs22/exceptions/LobbyNameConflictException.java

```diff
@@ -0,0 +1,4 @@
+package ch.uzh.ifi.hase.soprafs22.exceptions;
+
+public class LobbyNameConflictException extends Exception{
```

Comment:
Shouldn't this exception hold the name that caused the conflict?

Reply:
Sure, I'll add it